src/components/common/needsShutdown.tsx

```diff
@@ -249,6 +254,17 @@ export const VmNeedsShutdown = ({ vm } : { vm: VM }) => {
                     );
                 }) }
             </List>
+            <br />
+            <Button
+                size="sm"
+                variant="link"
+                onClick={() => {
+                    hide();
+                    Dialogs.show(<VmRestartDialog vm={vm} />);
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2376-d643e9ab-20251027-103129-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)